Js/main.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-env jquery */
+
+let myLibrary = [];
+const addNewBookButton = document.querySelector('button.newBook');
+
+function Book(title, author, pages, id) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+}
+
+function bookInfo(book) {
+  return `${book.title}, by ${book.author}, ${book.pages} pages`;
+}
+
+Book.prototype.info = bookInfo(this);
+
+Book.prototype.read = 'Not Read';
+
+function addBookToLibrary(title, author, pages, id) {
+  const newBook = new Book(title, author, pages, id);
+  myLibrary.push(newBook);
+}
```

Comment:
## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

Reply:
Converted